tests/translators/piglatinizer.test.js

```diff
@@ -0,0 +1,67 @@
+import * as piggyTranslator from '../../translators/piglatinizer';
+
+describe ('pig latinizer translator tests', () =>{
+
+  const job = {
+    "tus": [{
```

Comment:
BTW, in the filter, you use `translationUnits`, but in the translators, you use `tus`.  Do you want to be consistent?

Reply:
yeah, it's one of those inner conflicts between being concise and descriptive. Feel free to refactor!

Reply:
Fair enough.  Was just trying to understand intent.